server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java

```diff
@@ -336,6 +336,13 @@
       .defaultValue(64L * 1024L * 1024L)
       .withDescription("For multistorage, the event size exceed this value, flush data  to cold storage");
 
+  public static final ConfigOption<Long> FALLBACK_TIMES = ConfigOptions
+      .key("rss.server.fallback.times")
```

Comment:
rss.server.multipleStorage.failedTimesBeforeFallback?

Reply:
OK

Reply:
For configuration, we should use lower case letters.

Reply:
ok